mt.c

```diff
@@ -837,7 +837,11 @@ void color_print(int f_index, NEWWIN *win, proginfo *cur, char *string, regmatch
 		 * codes in that string which set a color those are stripped as well
 		 * the stripping part should be moved to a seperate function
 		 */
+    char *temp = malloc(strlen(string));
```

Comment:
Could this cause a memory leak? When is the allocated memory deallocated?

Reply:
Yeah, you're right. I had a test fork and another one to push commits from, because I actually had to edit the header files to get the program building on Arch Linux, and so made a mistake copying the changes from the test source into the source to be pushed. My bad.